src/components/WebexAvatar/WebexAvatar.stories.js

```diff
@@ -1,6 +1,28 @@
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 
+import PeopleJSONAdapter from '../../adapters/PeopleJSONAdapter';
+import {personStatuses} from '../../adapters/PeopleAdapter';
+import data, {ID} from '../../data/person';
+
 import WebexAvatar from './WebexAvatar';
 
-storiesOf('Webex Avatar', module).add('Avatar with title', () => <WebexAvatar title="Tom Smith" />);
+// Setup for the stories
+const person = {};
+
+person[ID] = data;
```

Comment:
Should we build a people.json data file instead of person.json? I was considering doing so with the PeopleJSONAdapter implementation so it would line up one-to-one data:adapter

Reply:
yeah, I'm going to do that. its cleaner